protos/caw.proto

```diff
@@ -65,8 +65,10 @@ message ProfileReply {
 
 message StreamRequest {
   string hashtag = 1;
+  string username = 2;
```

Comment:
I just want to make sure project specifications don't require a username (because I didn't have one for my project), I assumed it was optional

Reply:
Yup, it is not required. Although in phase 1 it was specified that only registered users should be able to perform any operations. If you have checked whether the user is registered some way or another I think you should be good. (Adam wasn't sure if it was required that the user streaming is registered, but better safe than sorry).